rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/telemetry/TbMsgDeleteAttributesNode.java

```diff
@@ -41,9 +41,10 @@
         configClazz = TbMsgDeleteAttributesNodeConfiguration.class,
         nodeDescription = "Delete attributes for Message Originator.",
         nodeDetails = "Attempt to remove attributes by selected keys. If msg originator doesn't have an attribute with " +
-                " a key selected in the configuration, it will be ignored. If delete operation is completed successfully, " +
-                " rule node will send the \"Attributes Deleted\" event to the root chain of the message originator and " +
-                " send the incoming message via <b>Success</b> chain, otherwise, <b>Failure</b> chain is used.",
+                " a key selected in the configuration, it will be ignored.<br><br>" +
+                "Output connections: if delete operation is completed successfully, rule node will send " +
+                "the <b>Attributes Deleted</b> event to the root chain of the message originator and send " +
+                "the incoming message via <code>Success</code> chain, otherwise <code>Failure</code> chain is used.",
```

Comment:
Let's change this to the way how we define connections in other nodes. 

Success - some description.
Failure - some description.

Reply:
Note: send of the <b>Attributes Deleted</b> event happens only if the checkbox is enabled.